core/domain/exp_services_test.py

```diff
@@ -1198,7 +1198,7 @@ class ZipFileExportUnitTests(ExplorationServicesUnitTests):
 states_schema_version: %d
 tags: []
 title: A title
-""" % (
+"""%(
```

Comment:
Please do not remove this space. This is not a docstring. Instead modify the check so that it can ignore this. Similar for the line below.

Reply:
Thanks for the suggestion. I will modify the check